sdk/appcenter/src/main/java/com/microsoft/appcenter/channel/DefaultChannel.java

```diff
@@ -175,6 +175,9 @@ public synchronized void addGroup(final String groupName, int maxLogsPerBatch, l
         if (mPersistence == null) {
             return;
         }
+        if (mGroupStates.containsKey(groupName)) {
```

Comment:
We discussed this with Clement and we will introduce a onGroupAdded callback instead in listener to register group. This will make sure pending logs of one collector are resumed on app restart even if no app center log enqueued (and will also remove the need for this check).

This would be a new work item but for the time being I would add a `/* TODO Remove this check once onGroupAdded callback is implemented. */`.